app/models/ability.rb

```diff
@@ -36,14 +48,14 @@ def initialize(user)
     # Any authenticated user can manage their own lessons.
     can %i[read create_copy update destroy], Lesson, user_id: user.id
```

Comment:
is this still meant to be here, we've excluded students but I wouldn't have thought we want a user that isn't a teacher or owner having the ability to do these things to a lesson though?

Reply:
I did mull this one over for sure but it's a legacy idea thing around having a bank of lessons detached from a school similar to "My Projects". I mulled over deleting all the old tests and bits too. Maybe I'll explicitly add `school_id: nil`

Reply:
there we go 😄 better than deleting well tested functionality we might use shortly